nvdocker/nvdocker.py

```diff
@@ -151,28 +151,28 @@ def exec_run(self, cid, cmd):
         return c.exec_run(cmd)
 
     @staticmethod
-    def list_gpus():
-        output = check_output(["nvidia-smi", "-L"]).decode("utf-8") 
-        regex = re.compile(r"GPU (?P<id>\d+):")
-        gpus = []
-        for line in output.strip().split("\n"):
-            m = regex.match(line)
-            assert m, "unable to parse " + line
-            gpus.append(int(m.group("id")))
+    def get_gpus():
```

Comment:
Rename if its going to be the parsed nvidia-smi output

Reply:
Back to list_gpus?

Reply:
probably gpu_info